Scheduler.js

```diff
@@ -0,0 +1,36 @@
+OS.Scheduler = {
+	
+    runNextProcess: function () {
+
+        var oNextProcess;
+        var readyProcesses;
+
+        readyProcesses = Processes.listOfProcesses.filter(function (process) {
+            return process.state == "Ready";
+        });
+
+
+        var readyProcessExists = readyProcesses != undefined && readyProcesses.length > 0;
+
+        if (readyProcessExists) {
+            oNextProcess = readyProcesses.reduce(function(oldest_process, current_process){
+                return oldest_process.last_acess < current_process.last_acess ? current_process : oldest_process;
```

Comment:
I think .last_acess need to be .last_access when trying to access the processes date data field.